test/chat_api/conversations_test.exs

```diff
@@ -41,11 +41,48 @@ defmodule ChatApi.ConversationsTest do
       different_customer = customer_fixture(different_account)
       _conversation = conversation_fixture(different_account, different_customer)
 
+      for _n <- 1..10,
+          do: message_fixture(account, conversation, %{sent_at: ~N[2020-11-14 20:50:26]})
+
       result_ids = Enum.map(Conversations.list_conversations_by_account(account.id), & &1.id)
 
       assert result_ids == [conversation.id]
     end
 
+    test "list_conversations_by_account_v2/2 returns all conversations with one message(most recent)",
```

Comment:
@WLSF looks like this test is failing?

Reply:
Yes, I was checking the reason... because it wasnt failing on the localhost...

Reply:
@reichert621 I've removed all the code about the list_conversations_v2, so I can work better on that in the background, and then we can back to it later when we found use for it to have a list of conversations with the top 10 messages, or something like that...